### PR TITLE
[REVIEW ONLY] 1.2.1 release notes

### PR DESCRIPTION
**Pull request by domcleal**
Please don't merge yet.

### CODE REVIEW

File: _includes/manuals/1.2/1.2_release_notes.md [hunk 1]
```diff
@@ -275,4 +275,69 @@ If you have problems with Passenger after upgrading the RPMs, please see [the SC
 * Host submit progress bar now present when hosted at non-root URL ([#2636](http://projects.theforeman.org/issues/2636))
 * Prevent 404 error when renaming an object with an invalid name ([#2404](http://projects.theforeman.org/issues/2404))
 
-*A full list of changes in 1.2 is available via the [Roadmap](http://theforeman.org/versions/show/24)*
+*A full list of changes in 1.2.0 is available via the [Roadmap](http://theforeman.org/versions/show/24)*
+
+
+### Release notes for 1.2.1
```
Comment (GregSutcliffe) [inline _includes/manuals/1.2/1.2_release_notes.md]: Stylistically, should 1.2.1 be above the 1.2.0 notes? I tend to expect that these days, but I have no idea if I'm in a minority or not ;)

**Comment by GregSutcliffe** [comment]
Style comment, content looks good :+1:

File: _includes/manuals/1.2/1.2_release_notes.md (hunk 1 above)
Comment (domcleal) [inline _includes/manuals/1.2/1.2_release_notes.md]: That seems fair, so between the 1.2 main release notes but before 1.2.0 specific?

Comment (skottler) [inline _includes/manuals/1.2/1.2_release_notes.md]: @domcleal yeah, that seems good.

Comment (domcleal) [inline _includes/manuals/1.2/1.2_release_notes.md]: updated

**Comment by lzap** [comment]
LGTM

File: _includes/manuals/1.2/1.2_release_notes.md [hunk 2]
```diff
@@ -99,7 +99,71 @@ Please see the end of the package sections for each platform for specific upgrad
...
+#### Provisioning templates
+
+* Add `repos` hook for repositories associated with a host and OS ([#2837](http://projects.theforeman.org/issues/2837))
+
+#### Puppet integration
+
+* Fix handle_ca error when no Puppet CA is assigned to a host ([#2693](http://projects.theforeman.org/issues/2693))
+* Prevent Puppet classes being assigned to a host before pressing "Submit" ([#2788](http://projects.theforeman.org/issues/2788))
+
+#### Security
```
Comment (GregSutcliffe) [inline _includes/manuals/1.2/1.2_release_notes.md]: Is there any value is a blank heading? It's a nitpick, feel free to ignore.

**Comment by GregSutcliffe** [comment]
minor nitpick, otherwise :+1:

File: _includes/manuals/1.2/1.2_release_notes.md (hunk 2 above)
Comment (domcleal) [inline _includes/manuals/1.2/1.2_release_notes.md]: removed, ta

**Pull request merged**